kube/resources/admin-service-account.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: airbyte-admin
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: airbyte-admin-role
+  namespace: default
+rules:
+  - apiGroups: ["*"]
+    resources: ["jobs", "pods", "pods/log", "pods/exec"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"] # over-permission for now
```

Comment:
which of these do we actually need? can we restrict it now or you want to keep it open becuase you are still iterating?

Reply:
I think we might be able to restrict it a bit more, but I'd prefer to overpermission until the kube implementation is more stable (even after this PR). How we launch jobs may change and it's really irritating to map those to permissions.